### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by Sherlockishappy**
Added assignment 2, kindly review. /Jing

### CODE REVIEW

File: CalcCurrentValue/Program.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            
+            Console.WriteLine("Nominal Amount:");           
+            int nominal = 0;
+            if (!Int32.TryParse(Console.ReadLine(), out nominal)) {
+                Console.WriteLine("Norminal amount need to be an integer number, try again:");
+                nominal = Int32.Parse(Console.ReadLine());
+            }
+
+            Console.WriteLine("Buy or Sell?");          
+            TransactionType BorS =(TransactionType)Enum.Parse(typeof(TransactionType), Console.ReadLine(),true);
```
Comment (d-okhrimenko) [inline CalcCurrentValue/Program.cs]: Please try rename variable BorS. It is better to use camelCasing for local variable names and use descriptive names.

File: CalcCurrentValue/Program.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            var currentValue = sign * nominal * tradePrice;
+            Console.WriteLine($"{BorS.ToString()} "+$"{nominal} trade "+$"with traded price = {tradePrice} means you will have to pay "+$"{currentValue} for the current value.");
+            //Calculate P&L for Sell transactions
+            var PL = plFactor * (tradePrice - originalPrice) * nominal;
+            Console.WriteLine($"You realized PL is: {PL}");
+
+            Console.ReadKey();
+        }
+
+        private enum TransactionType { Buy, Sell };
```
Comment (d-okhrimenko) [inline CalcCurrentValue/Program.cs:41]: Best practice for new types (enum is a new type) is to put them in a separate file.
For such small project is not an issue but in future consider moving of new types into new file.

**Review by d-okhrimenko — COMMENTED** [review]
Please use Ctrl + K, Ctrl + D to fix formatting in document. Few spaces are missing and indents in few lines are incorrect

**Review by d-okhrimenko — COMMENTED** [review]
Please use Ctrl + K, Ctrl + D to fix formatting in document. Few spaces are missing and indents in few lines are incorrect

File: CalcCurrentValue/Program.cs (hunk 1 above)
Comment (Sherlockishappy) [reply CalcCurrentValue/Program.cs]: it stands for 'Buy' or 'Sell' :-)

**Pull request merged**